docs/pages/versions/v48.0.0/sdk/notifications.mdx

```diff
@@ -221,7 +221,7 @@ export default function App() {
           const onReceiveURL = ({ url }: { url: string }) => listener(url);
 
           // Listen to incoming links from deep linking
-          Linking.addEventListener('url', onReceiveURL);
+          const subEvent = Linking.addEventListener('url', onReceiveURL);
```

Comment:
```suggestion
          const eventListenerSubscription = Linking.addEventListener('url', onReceiveURL);
```
I'd suggest changing the name of the variable to something meaningful that represents the event listener.